File: src/VSServiceStack/ProjectTemplates/FSharpSelfHost/EmptySelfHost/AppHost.fs
namespace $safeprojectname$
    open ServiceStack
    open ServiceStack.Common
    open System
    open $safeprojectname$.ServiceInterface

    type AppHost() = 
        inherit AppSelfHostBase("$safeprojectname$", typeof<MyServices>.Assembly)
        override this.Configure container =
            //Add plugins
            //this.Plugins.Add(new PostmanFeature())
            //this.Plugins.Add(new CorsFeature())
            ignore()

File: src/ServiceStackVS/ProjectTemplates/AngularJS/ServiceStackAngularJS/AppHost.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Reflection;
using System.Web;
using Funq;
using $saferootprojectname$.ServiceInterface;
using ServiceStack.Razor;
using ServiceStack;

namespace $safeprojectname$
{
    public class AppHost : AppHostBase
    {
        /// <summary>
        /// Base constructor requires a Name and Assembly where web service implementation is located
        /// </summary>
        public AppHost()
            : base("$safeprojectname$", typeof(MyServices).Assembly) {}

        /// <summary>
        /// Application specific configuration
        /// This method should initialize any IoC resources utilized by your web service classes.
        /// </summary>
        public override void Configure(Container container)
        {
            //Config examples
            //this.Plugins.Add(new PostmanFeature());
            //this.Plugins.Add(new CorsFeature());
            
            this.Plugins.Add(new RazorFormat());
        }
    }
}